### PR TITLE
Feature/(flw 1651) export wallet final

### PR DESCRIPTION
**Pull request by bnsports**

### CODE REVIEW

File: fearless/Modules/Export/ExportGenericView/ExportGenericViewController.swift [hunk 1]
```diff
@@ -141,6 +138,18 @@ final class ExportGenericViewController: UIViewController, ImportantViewProtocol
...
             views.append(newOptionView)
+
+            if exportViewModel.option == .keystore {
+                if exportViewModel.ethereumBased {
+                    setupExportEthereumButton()
+                } else {
+                    setupExportSubstrateButton()
+                }
+            } else {
+                if mainOptionTitle != nil {
```
Comment (DRadmir) [inline fearless/Modules/Export/ExportGenericView/ExportGenericViewController.swift:149]: You can remove one level. Change to `else if mainOptiontitle != nil`

File: fearless/Modules/Export/AccountExportPassword/AccountExportPasswordInteractor.swift [hunk 2]
```diff
@@ -48,25 +118,35 @@ extension AccountExportPasswordInteractor: AccountExportPasswordInteractorInputP
...
                     }
+                    self.operationManager.enqueue(operations: [genesisOperation], in: .transient)
+                } else {
+                    self.createExportOperation(
+                        address: address,
+                        password: password,
+                        chain: chain,
+                        chainAccount: response,
+                        metaId: metaAccount.metaId,
+                        genesisHash: ""
```
Comment (DRadmir) [inline fearless/Modules/Export/AccountExportPassword/AccountExportPasswordInteractor.swift:147]: change "" to nil please

File: fearless/Modules/Export/ExportGenericView/ExportGenericViewController.swift [hunk 3]
```diff
@@ -203,6 +220,40 @@ extension ExportGenericViewController {
...
+            make.height.equalTo(UIConstants.actionHeight)
+        }
+
+        button.addTarget(
+            self,
+            action: #selector(exportSubstrateButtonClicked),
+            for: .touchUpInside
+        )
+
+        button.imageWithTitleView?.title = R.string.localizable.exportSubstrateTitle(preferredLanguages: selectedLocale.rLanguages)
```
Comment (DRadmir) [inline fearless/Modules/Export/ExportGenericView/ExportGenericViewController.swift:237]: Need to implement `Localizable` protocol for ExportGenericViewController

File: fearless/Modules/Export/ExportGenericView/ExportGenericViewController.swift [hunk 4]
```diff
@@ -203,6 +220,40 @@ extension ExportGenericViewController {
...
+            make.height.equalTo(UIConstants.actionHeight)
+        }
+
+        button.addTarget(
+            self,
+            action: #selector(exportEthereumButtonClicked),
+            for: .touchUpInside
+        )
+
+        button.imageWithTitleView?.title = R.string.localizable.exportEthereumTitle(preferredLanguages: selectedLocale.rLanguages)
```
Comment (DRadmir) [inline fearless/Modules/Export/ExportGenericView/ExportGenericViewController.swift:254]: Need move to `setupLocalization()`

File: fearless/Modules/Export/ExportMnemonic/ExportMnemonicInteractor.swift [hunk 5]
```diff
@@ -28,6 +28,8 @@ final class ExportMnemonicInteractor {
 }
 
 extension ExportMnemonicInteractor: ExportMnemonicInteractorInputProtocol {
+    func fetchExportDataForWallet(_: MetaAccountModel) {}
```
Comment (DRadmir) [inline fearless/Modules/Export/ExportMnemonic/ExportMnemonicInteractor.swift:31]: this func not have implementation?

File: fearless/Modules/Export/ExportMnemonic/ExportMnemonicPresenter.swift [hunk 6]
```diff
@@ -7,55 +7,62 @@ final class ExportMnemonicPresenter {
...
+//                    data.mnemonic.toString(),
+//                    preferredLanguages: locale.rLanguages
+//                )
+//        }
+//
+//        wireframe.share(source: TextSharingSource(message: text), from: view) { [weak self] completed in
+//            if completed {
+//                self?.wireframe.close(view: self?.view)
+//            }
+//        }
```
Comment (DRadmir) [inline fearless/Modules/Export/ExportMnemonic/ExportMnemonicPresenter.swift:50]: comment

File: fearless/Modules/Export/ExportMnemonic/ExportMnemonicPresenter.swift [hunk 7]
```diff
@@ -7,55 +7,62 @@ final class ExportMnemonicPresenter {
...
+
+    func didTapExportEthereumButton() {}
+
     func setup() {
-        interactor.fetchExportDataForAddress(address, chain: chain)
+        switch flow {
+        case let .single(chain, address):
+            interactor.fetchExportDataForAddress(address, chain: chain)
+        case let .multiple(account):
+            break
```
Comment (DRadmir) [inline fearless/Modules/Export/ExportMnemonic/ExportMnemonicPresenter.swift:64]: Mark `//TODO:` please

**Review by Viscount88 — APPROVED** [review]

**Pull request merged**